en_ewt-ud-test.conllu

```diff
@@ -28467,7 +28467,7 @@
 7	emergency	emergency	NOUN	NN	Number=Sing	8	compound	8:compound	_
 8	plumbers	plumber	NOUN	NNS	Number=Plur	5	obj	5:obj|10:nsubj	_
 9	who	who	PRON	WP	PronType=Rel	10	nsubj	8:ref	_
-10	visted	visit	VERB	VBD	Mood=Ind|Tense=Past|Typo=Yes|VerbForm=Fin	8	acl:relcl	8:acl:relcl	CorrectForm=visited
+10	visted	visit	VERB	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|Typo=Yes|VerbForm=Fin	8	acl:relcl	8:acl:relcl	CorrectForm=visited
```

Comment:
An interesting case: subject is a relative pronoun. I suppose this should be `Number=Plur` because the antecedent is plural? In the present tense it would matter for agreement:

- the plumber who visits / the plumbers who visit

Reply:
right, it should be plural, there are probably a few other cases like these.

What do you suggest for "The team who work there are helpfull," (line 31963), should "work" be  "plural" (probably because 'are' is plural, even though the subject is singular. Those are the kind of things that should be decided manually.

Reply:
Yes I would say that is plural because "team" is being interpreted collectively.

Reply:
Hello

I have reviewed the "dev" and "test" for sentence with a relative pronoun as subject. It seems that these modifications have been pushed OK (see screen shoot).
I wanted to send a Pull Request for these, but I cannot find the "button" to do it...
![screenshot](https://user-images.githubusercontent.com/6200743/125881744-056ca925-b776-45e4-a4be-c1f221816132.jpg)

Do you see these modifications ?
thanks
Guy

Reply:
Yes the modifications look good. They are modifications to the original pull request since I haven't merged it yet.